private-templates-service/client/src/components/NavBar/NavBar.tsx

```diff
@@ -1,77 +1,19 @@
-import React from "react";
-import {
-  Collapse,
-  Container,
-  Navbar,
-  NavbarToggler,
-  NavbarBrand,
-  Nav
-} from "reactstrap";
-
-// Redux
+import React, {useState, useEffect} from 'react';
 import { connect } from "react-redux";
 import { RootState } from "../../store/rootReducer";
-
-// Components
-import { UserType } from "../../store/auth/types";
-import AuthNavItem from "./AuthNavItem";
-
-// Styles
-import "@fortawesome/fontawesome-free/css/all.css";
-
-interface NavBarProps {
-  authButtonMethod: () => void;
-  isAuthenticated: boolean;
-  user?: UserType;
-}
-
-interface State {
-  isOpen: boolean;
-}
-
-const mapStateToProps = (state: RootState) => {
-  return {
-    isAuthenticated: state.auth.isAuthenticated,
-    user: state.auth.user
-  };
-};
-
-class NavBar extends React.Component<NavBarProps, State> {
-  constructor(props: NavBarProps) {
-    super(props);
-
-    this.state = {
-      isOpen: false
-    };
-  }
-
-  toggle = () => {
-    this.setState({
-      isOpen: !this.state.isOpen
-    });
-  }
-
-  render() {
-    return (
-      <div>
-        <Navbar color="dark" dark expand="md" fixed="top">
-          <Container>
-            <NavbarBrand href="/">Admin Portal</NavbarBrand>
-            <NavbarToggler onClick={this.toggle} />
-            <Collapse isOpen={this.state.isOpen} navbar>
-              <Nav className="ml-auto" navbar>
-                <AuthNavItem
-                  authButtonMethod={
-                    this.props.authButtonMethod
-                  }
-                />
-              </Nav>
-            </Collapse>
-          </Container>
-        </Navbar>
-      </div>
-    );
-  }
+import SearchBar from "./SearchBar/SearchBar";
```

Comment:
can you create an `index.tsx` file in `./SearchBar` so that you can just `import SearchBar from "./SearchBar"`.

see `private-templates-service\client\src\components\Dashboard\index.ts` for an example.